### DISCUSSION TITLE
Accelerate with AI: Build your own GitHub Copilot Extension

### DISCUSSION THREAD

**Discussion by alexisabril** [community/community#142312] [Other Feature Feedback, Questions, & Ideas]
Universe Sandbox Workshop: [AI1894S](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1722542471073001sb4S) - Accelerate with AI: Build your own GitHub Copilot Extension

> AI is a critical tool for accelerating software development today. That said, AI tooling isn't always tailored to your business use case. In this session with GitHub's Alexis Abril, senior partner engineer, and Dimitrios Philliou, product manager, GitHub Copilot Extensions, you'll learn how to extend GitHub Copilot and begin to develop your own tailor-made AI solutions.

Thank you for interest and/or attending our workshop on building Copilot Extensions! Feel free to comment here for feedback, questions, and any and all discussions around Copilot Extensibility! If you are attending Universe, be sure to stop by the Copilot Extensibility booth in the Gateway Pavilion.

The [Copilot Extensions Organization](https://github.com/copilot-extensions) is a great place for additional resources.

Hi folks and thank you for your interest in this workshop! The workshop materials and all steps we followed to create your own Copilot App are now available [here](https://github.com/github-technology-partners/AI1894S-Accelerate-with-AI). Feel free to use this discussion to continue the thread with any questions!

Let's build together!

**Comment by Shahramgithib** [comment]
هوش مصنوعی باورود به همه ی عرصه های زندگی در همه ی جنبه های مختلف از هنر تا جنگ وارد شده و هر روز جذابیت هایش  بیشتر میشود

**Comment by JoshuaGruber** [comment]
The session was full, is there a recording or presentation available?

**Reply by anikettt2499** [reply]
Yes, you can take a look at this: https://www.youtube.com/watch?v=5ov2NYBdGSw .

**Reply by nttd-ushijimay** [reply]
That video looks like private and couldn't see it. Would you make it publich?

**Reply by anikettt2499** [reply]
I think it should be public. you can search for **GitHub Universe 2024 - Day 1 Tuesday, Oct 29** on youtube:)

**Reply by alexisabril** [reply]
Hi @JoshuaGruber! Apologies as I do know our session filled up quickly! I've posted our materials here and feel free to respond with any questions! https://github.com/github-technology-partners/AI1894S-Accelerate-with-AI
